bu-slideshow.php

```diff
@@ -693,22 +705,8 @@ static public function delete_slideshow_ajax() {
 	 * @return int
 	 */
 	static public function delete_slideshow($id) {
-		if (!self::slideshow_exists($id)) {
-			return;
-		}
-		
-		$all_slideshows = self::get_slideshows();
-		
-		unset($all_slideshows[$id]);
-
-		if( version_compare( get_bloginfo('version'), '3.6', '>=') ){
-			update_option(BU_Slideshow::$meta_key, $all_slideshows);
-		} else {
-			delete_option(BU_Slideshow::$meta_key);
-			add_option(BU_Slideshow::$meta_key, $all_slideshows);
-		}
-
-		return 1;
+		$id = self::slideshow_maybe_translate_id( $id );
```

Comment:
Pulling `slideshow_maybe_translate_id()` out of slideshow-selector.php (as it was unnecessary). Adding `get_slideshow()` to the functions like `delete_slideshow()` & `slideshow_exists()` would make them a bit heavy, in my opinion.